CMakeLists.txt

```diff
@@ -334,6 +334,7 @@ if(WIN32)
 
   SET(PSAPI_LIBS "psapi.lib")
 
+  set(USE_EXACT_ALLOC_COUNT)
```

Comment:
Could you use `set(USE_EXACT_ALLOC_COUNT TRUE)` like `USE_SELECT`?

Reply:
I've fixed.